web/views/layout_view.ex

```diff
@@ -2,6 +2,7 @@ defmodule Appointments.LayoutView do
   use Appointments.Web, :view
 
   def active?(conn, path) do
-    if conn.request_path == "/#{path}", do: "active", else: ''
+    [_ | [resource | _]] = String.split(conn.request_path, "/")
```

Comment:
could this possibly cause an unhandled error if `conn.request_path` has a different format than we expect, f.x. `/`